Formula/libvisual.rb

```diff
@@ -0,0 +1,110 @@
+class Libvisual < Formula
+  desc "Audio Visualization tool and library"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-0.4.2/libvisual-0.4.2.tar.gz"
+  sha256 "63085fd9835c42c9399ea6bb13a7ebd4b1547ace75c4595ce8e9759512bd998a"
+  license all_of: ["GPL-2.0-or-later", "GPL-3.0-or-later", "LGPL-2.1-or-later"]
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "xorg-server" => :test
+
+  depends_on "jack"
+  depends_on "portaudio"
+  depends_on "projectm"
+  depends_on "sdl12-compat"
+
+  uses_from_macos "bison" => :build
+
+  on_linux do
+    depends_on "bison" => :build
+    depends_on "alsa-lib"
+    depends_on "libx11"
+    depends_on "libxext"
+    depends_on "libxrender"
+    depends_on "mesa"
+    depends_on "mesa-glu"
+    depends_on "pulseaudio"
+  end
+
+  resource "libvisual-plugins" do
+    url "https://github.com/Libvisual/libvisual/releases/download/libvisual-plugins-0.4.2/libvisual-plugins-0.4.2.tar.gz"
+    sha256 "55988403682b180d0de5e6082f804f3cf066d9a08e887b10eb6a315eb40d9f87"
+  end
+
+  resource "libvisual-projectm" do
+    url "https://github.com/projectM-visualizer/frontend-libvisual-plug-in/archive/refs/tags/v2.1.1.tar.gz"
+    sha256 "eb8269c2a923546600d3f40ff90c011f03a215847215ee8bef44bfae305b4dd7"
+  end
+
+  def install
+    system "./configure", *std_configure_args
+    system "make"
+    system "make", "install"
+
+    ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
+
+    resource("libvisual-plugins").stage do
+      configure_args = [
+        "--disable-gdkpixbuf-plugin",
+        "--disable-gforce",
+        "--disable-gstreamer-plugin",
+        *std_configure_args,
+      ]
+      system "./configure", *configure_args
+      system "make"
+      system "make", "install"
+    end
+
+    resource("libvisual-projectm").stage do
+      system "cmake", ".", *std_cmake_args
+      system "make"
+      system "make", "install"
```

Comment:
Let's use the template suggested by `brew create --cmake`. (You can change the name of the build directory below if needed.)
```suggestion
      system "cmake", "-S", ".", "-B", "build", *std_cmake_args
      system "cmake", "--build", "build"
      system "cmake", "--install", "build"
```

Reply:
@carlocab so standard out-of-source build. Can do. Done.